phone-book.csv

```diff
@@ -0,0 +1 @@
+Jan Kowalski, 123123123
```

Comment:
I'm pretty sure we dont need to commit this file since it is created during app runtime.